docker-compose.override.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.4"
```

Comment:
It looks like all features you use are supported in version 3.8, so you could keep 3.4 to support a larger set of Docker Compose versions.

Reply:
Sure you are right, I didn't think about